packages/fuel-gauge/src/transaction-response.test.ts

```diff
@@ -247,42 +246,46 @@ describe('TransactionResponse', () => {
     cleanup();
   });
 
-  it('should throw error for a SqueezedOut status update [submitAndAwait]', async () => {
-    const { cleanup, ip, port } = await launchNode({
-      args: ['--poa-instant', 'false', '--poa-interval-period', '1s', '--tx-pool-ttl', '200ms'],
-      loggingEnabled: false,
-    });
-    const nodeProvider = await Provider.create(`http://${ip}:${port}/v1/graphql`);
-
-    const genesisWallet = new WalletUnlocked(
-      process.env.GENESIS_SECRET || randomBytes(32),
-      nodeProvider
-    );
-
-    const request = new ScriptTransactionRequest();
-
-    request.addCoinOutput(Wallet.generate(), 100, baseAssetId);
-
-    const txCost = await genesisWallet.provider.getTransactionCost(request, {
-      signatureCallback: (tx) => tx.addAccountWitnesses(genesisWallet),
-    });
-
-    request.gasLimit = txCost.gasUsed;
-    request.maxFee = txCost.maxFee;
-
-    await genesisWallet.fund(request, txCost);
-
-    request.updateWitnessByOwner(
-      genesisWallet.address,
-      await genesisWallet.signTransaction(request)
-    );
-
-    await expectToThrowFuelError(
-      async () => {
-        await nodeProvider.sendTransaction(request, { awaitExecution: true });
-      },
-      { code: ErrorCode.TRANSACTION_SQUEEZED_OUT }
-    );
-    cleanup();
-  });
+  it(
+    'should throw error for a SqueezedOut status update [submitAndAwait]',
+    async () => {
+      const { cleanup, ip, port } = await launchNode({
+        args: ['--poa-instant', 'false', '--poa-interval-period', '4s', '--tx-pool-ttl', '1s'],
+        loggingEnabled: false,
+      });
+      const nodeProvider = await Provider.create(`http://${ip}:${port}/v1/graphql`);
+
+      const genesisWallet = new WalletUnlocked(
+        process.env.GENESIS_SECRET || randomBytes(32),
+        nodeProvider
+      );
+
+      const request = new ScriptTransactionRequest();
+
+      request.addCoinOutput(Wallet.generate(), 100, baseAssetId);
+
+      const txCost = await genesisWallet.provider.getTransactionCost(request, {
+        signatureCallback: (tx) => tx.addAccountWitnesses(genesisWallet),
+      });
+
+      request.gasLimit = txCost.gasUsed;
+      request.maxFee = txCost.maxFee;
+
+      await genesisWallet.fund(request, txCost);
+
+      request.updateWitnessByOwner(
+        genesisWallet.address,
+        await genesisWallet.signTransaction(request)
+      );
+
+      await expectToThrowFuelError(
+        async () => {
+          await nodeProvider.sendTransaction(request, { awaitExecution: true });
+        },
+        { code: ErrorCode.TRANSACTION_SQUEEZED_OUT }
+      );
+      cleanup();
+    },
+    { retry: 10 }
```

Comment:
So the only way to avoid this test being flaky is to retry it?

Reply:
I don't know the underlying reason for its flakiness so, afaik, yes. The good thing is that if the test is failing due to valid reasons then it'll fail after the retries anyways.

Reply:
It looks like a temporary solution that could hide the real [bigger?] issue.

It could have been valid to have an explanation and a TODO for future review.